packages/webpack-cli/lib/groups/AdvancedGroup.js

```diff
@@ -57,6 +57,12 @@ class AdvancedGroup extends GroupHelper {
                 options.plugins = [hotModuleVal];
```

Comment:
Could you also create a unit test inside `AdvancedGroup.test.js` file? Similar to `ZeroConfigGroup.test.js`